cmux-tui/scripts/smoke-tui.py

```
@@ -703,25 +713,25 @@ def assert_snapshot_parser_controls():
+write_all(fd, b"\x03")
 drain(0.4)
 
 # Drag-select the marker text: press, drag, release (SGR mouse, 1-based).
 # Pane content starts at column 24 (sidebar 22 + left border 1; SGR
 # 1-based) and row offset 1 for the top border. On release the TUI must
 # copy the selection to the host clipboard as an OSC 52 sequence.
-os.write(fd, b"clear; printf 'smoke-marker-%s\\n' ok\r")
+write_all(fd, b"clear; printf 'smoke-marker-%s\\n' ok\r")
 wait_screen_contains(surface_id, "smoke-marker-ok")
 lines = rpc({"id": 100, "cmd": "read-screen", "surface": surface_id})["data"]["text"].splitlines()
 vrow = next(i for i, l in enumerate(lines) if "smoke-marker-ok" in l)
```

Ruff (0.16.1)

[error] Ambiguous variable name: `l`

(E741)